_layouts/home.html

```diff
@@ -71,13 +71,19 @@ <h2>Formats</h2>
           <div class="card-media">{% include icon__format--video.svg %}</div>
         </div>
       </div>
+      <div class="card text-center">
+        <div class="card-body">
+          <a class="card-title stretched-link" href="/formats/ctv.html">Connected TV</a>
+          <div class="card-media"><img src="/assets/images/homepage/ctv_grey.png" alt="ctv"></div>
+        </div>
+      </div>
       <div class="card text-center">
         <div class="card-body">
           <a class="card-title stretched-link" href="/formats/amp.html">AMP</a>
           <div class="card-media">{% include icon__format--amp.svg %}</div>
         </div>
       </div>
-      <div class="w-100 d-none d-md-block d-xl-none"></div>
+      <!--div class="w-100 d-none d-md-block d-xl-none"></div-->
```

Comment:
Why commented out and not removed?

Reply:
Sorry - this was me trying to fix a formatting issue. Will remove the comments.